sdks/java/testing/jpms-tests/build.gradle

```diff
@@ -84,6 +84,7 @@ for (String runner : testRunnerClass.keySet()) {
     ]
     if ("dataflowRunnerIntegrationTest".equals(runner)) {
       testPipelineOptions.add("--region=${gcpRegion}")
+      testPipelineOptions.add("--experiments=use_staged_dataflow_worker_jar")
```

Comment:
in L66, consider add dataflowRunnerIntegrationTest project(":sdks:java:harness"), currently SDK harness in class path is due to transient dependency

runners-cloud-dataflow-java -> sdks-io-google-cloud-platform -> sdk-java-harness, introduced in https://github.com/apache/beam/commit/ba17f369e9304d28becc2a49783ba3a6acb5abac#r159238575 and shouldn't be there imo